tensorbay/sensor/intrinsics.py

```diff
@@ -487,6 +743,27 @@ def project(self, point: Sequence[float], is_fisheye: bool = False) -> Vector2D:
         Returns:
             The coordinates on the pixel plane where the point is projected to.
 
+        Example:
```

Comment:
There should be some adjustments same as the ones mentioned in the comment of ``distortion_coefficients.distort()``.